### DISCUSSION TITLE
Where restore is done

### DISCUSSION THREAD

**Discussion by boubou92** [community/community#22977] [New to GitHub]
<p>Hello Guys,</p>
<p>Sorry i’am NEW in Gitlab, i’am using gitlab CE. I have done restore from backup and I’am looking for the path where the restore is done.</p>
<p>I’am using omnibus package for the installation and doing rpm -i. </p>
<p>I’am using 11.02 version, I have modified backup folder but I cannot found the folder where the restore is done.</p>
<p>Is there any temp files created when the restore is in progress ?</p>
<p>I’am asking because the / is limited and I cannot extend it but I can mount new disk in the restore path.</p>
<p>Thanks for help</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/boubou92">@boubou92</a>,</p>
<p>You’re on the GitHub forum, not GitLab :slight_smile: You may have more success finding an answer in a GitLab specific place, see <a href="https://about.gitlab.com/support/" rel="nofollow noopener">https://about.gitlab.com/support/</a> and <a href="https://about.gitlab.com/get-help/" rel="nofollow noopener">https://about.gitlab.com/get-help/</a> to find some</p>

**Comment by boubou92** [comment]
<p>Hoo my bad, i’am sorry :smiley:</p>
